modules/ROOT/pages/questions/question-06.adoc

```diff
@@ -0,0 +1,12 @@
+6. `paste` no longer supports Microsoft Word.
+
+   What precisely does this mean? If I copy styled text from Microsoft
```

Comment:
@EkimChau

Reply:
We removed specific handling for Microsoft Word in this release. Content can still be pasted, and in fact it will appear with full formatting, but the HTML generated will be a mess. That's what our `paste` handling does; it makes the HTML nice.

There are two things worth mentioning in the release notes or migration guide:
- customers who wish to paste content from Microsoft Word should look at our premium paste offering
- we are offering the word import functionality to the community, if anyone can commit to maintaining it we will link to their plugin in our migration guide https://github.com/tinymce/tinymce/discussions/7487